README.md

```diff
@@ -360,6 +367,72 @@ The [`FairplayConfig`](https://github.com/bitmovin/bitmovin-player-react-native/
 
 Also, don't forget to check out the [example](https://github.com/bitmovin/bitmovin-player-react-native/tree/development/example) app for a complete iOS/Android [DRM example](https://github.com/bitmovin/bitmovin-player-react-native/blob/development/example/src/screens/BasicDRMPlayback.tsx).
 
+### Adding external subtitle tracks
+
+Usually, subtitle tracks are provided in the manifest of your content (see [Enconding Manifests API](https://bitmovin.com/docs/encoding/api-reference/sections/manifests) for more information). And if they are provided this way, the player already recognizes them and show them in the subtitles selection menu without any further configuration.
+
+Otherwise, it's also possible to add external tracks via the subtitle API:
+
+```typescript
+import { Platform } from 'react-native';
+import {
+  SourceConfig,
+  SourceType,
+  SubtitleFormat,
+} from 'bitmovin-player-react-native';
+
+// Source config with an external subtitle track.
+const config: SourceConfig = {
+  url:
+    Platform.OS === 'ios'
+      ? 'https://bitmovin-a.akamaihd.net/content/sintel/hls/playlist.m3u8'
+      : 'https://bitmovin-a.akamaihd.net/content/sintel/sintel.mpd',
+  type: Platform.OS === 'ios' ? SourceType.HLS : SourceType.DASH,
+  poster: 'https://bitmovin-a.akamaihd.net/content/sintel/poster.png',
+  // External subtitle tracks list to be added to this source.
+  addSubtitleTracks: [
+    // You can select 'Custom English' in the subtitles menu.
+    {
+      // The URL of the subtitle file. Required.
+      url: 'https://bitdash-a.akamaihd.net/content/sintel/subtitles/subtitles_en.vtt',
+      // External file format.
+      // Supports `.vtt`, `.ttml` and `.cea` extensions.
+      //
+      // This option can be left empty since the player automatically recognizes the format
+      // from the provided url most of the time.
+      format: SubtitleFormat.VTT,
+      // Label for this track shown under the selection menu. Required.
+      label: 'Custom English',
+      // The language tag for this track. Required.
```

Comment:
Do you think we should do a verification step (maybe in JS) to validate this input and throw an error earlier?

There's already an [npm package](https://www.npmjs.com/package/bcp-47) to deal with bcp-47 tags and we could use specifically its [`warning`](https://www.npmjs.com/package/bcp-47#function-warningreason-code-offset) feature to give user-friendly messages about problems with the provided tags. What do you think?

Reply:
Regarding the suggested change on README, it's done 3722767c82d14b3bdde6aac097f55a50e5bf493e.

Reply:
Sorry for the late reply, I was out-of-office.

That additional verification sounds like a great addition, but I think it is out of scope for now as we are adding basic subtitle support right now. It would be a great improvement later.